test/unit/stats.js

```diff
@@ -0,0 +1,793 @@
+// TODO Add before each that reseeds the DB
+// Set the env to development
+process.env.NODE_ENV = 'development';
+
+// Require the testing dependencies
+var chai = require('chai');
+var assert = chai.assert;
+
```

Comment:
// Require seed to reset database before each test
const seed = require('../../lib/utils').seed;

Add that ^^^